ui/src/app/app-routing.module.ts

```diff
@@ -30,7 +30,7 @@ import { JobTemplatesHomeComponent } from './components/admin/job-templates/job-
 import { JobTemplateShowComponent } from './components/admin/job-templates/job-template-show/job-template-show.component';
 
 const routes: Routes = [
-  { path: routeNames.DEFAULT, redirectTo: routeNames.WORKFLOWS, pathMatch: 'full', canActivate: [AuthGuardService] },
```

Comment:
Why was `canActivate: [AuthGuardService]` removed?

Reply:
It is redundant, canActivate will kick off after redirect

Reply:
Update:
I reverted my revert because 

Error: Invalid configuration of route '': redirectTo and canActivate cannot be used together. Redirects happen before activation so canActivate will never be executed.